idd/Energy+.idd.in

```diff
@@ -46210,7 +46223,7 @@ Coil:Heating:DX:SingleSpeed,
        \note wb = wet-bulb temperature (C) of air entering the indoor coil
        \note oat = outdoor air dry-bulb temperature (C)
        \note only required if ReverseCycle defrost strategy is specified
-  N5 , \field Minimum Outdoor Dry-Bulb Temperature for Compressor Operation
+  N5 , \field Minimum Outdoor Dry-Bulb Temperature for Compressor Operation {C}
        \type real
        \minimum -20.0
        \default -8.0
```

Comment:
For heating, I guess so.  But why a min for any of these coil types? Cooling could go down to -25C?